### PR TITLE
fix(DB/Texts): Added texts to Halazzi.

### LINKED ISSUE #8744: [Instance/Zul'A man]Halazzi dont have texts in DB
###What client do you play on?
zhCN

###Faction
 Alliance

###Content Phase:
70

### Current Behaviour

when you combat with Halazzi, he didnot say anything texts.

and DBErrors LOGS show :

CreatureTextMgr: Could not find Text for Creature(Halazzi) Entry 23577 in 'creature_text' table. Ignoring.
CreatureTextMgr: Could not find Text for Creature(Halazzi) Entry 23577 in 'creature_text' table. Ignoring.

### Expected Blizzlike Behaviour

he will say some text when he aggro, split, combine, melee, berserk.

### Source

https://tbc.wowhead.com/npc=23577/halazzi#sounds

### Steps to reproduce the problem

1. .go c 86195
2. combat with Halazzi
3. he didnot say anything

### Extra Notes

no

### AC rev. hash/commit

04143658ab406106ff2650e944592d1421987e21

### Operating system

WIN10

### Custom changes or Modules

account achi
ah bot
autobalance
eluna
solo lfg

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #8744

## Issues Addressed:

- Closes #8744

## Tests Performed:

- Not tested.

## How to Test the Changes:

`.go c 86195`
combat with Halazzi

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1670753866467974200.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--
+DELETE FROM `creature_text` WHERE `CreatureID`=23577;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(23577,0,0,'Get on ya knees and bow.... to da fang and claw!',14,0,100,0,0,12020,0,0,'Halazzi - SAY_AGGRO'),
+(23577,1,0,'You can\t fight da power!',14,0,100,0,0,12026,0,0,'Halazzi - SAY_PLAYER_KILL'),
+(23577,1,1,'Ya all gonna fail!',14,0,100,0,0,12027,0,0,'Halazzi - SAY_PLAYER_KILL'),
+(23577,2,0,'Me gonna carve ya now!',14,0,100,0,0,12023,0,0,'Halazzi - SAY_MELEE'),
+(23577,2,1,'You gonna leave in pieces!',14,0,100,0,0,12024,0,0,'Halazzi - SAY_MELEE'),
+(23577,3,0,'I fight wit\ untamed spirit....',14,0,100,0,0,12021,0,0,'Halazzi - SAY_SPLIT'),
+(23577,4,0,'Spirit, come back to me!',14,0,100,0,0,12022,0,0,'Halazzi - SAY_COMBINE'),
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1670753866467974200.sql:10]: in Core this is SAY_MERGE, update the comment accordingly.

File: data/sql/updates/pending_db_world/rev_1670753866467974200.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--
+DELETE FROM `creature_text` WHERE `CreatureID`=23577;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(23577,0,0,'Get on ya knees and bow.... to da fang and claw!',14,0,100,0,0,12020,0,0,'Halazzi - SAY_AGGRO'),
+(23577,1,0,'You can\t fight da power!',14,0,100,0,0,12026,0,0,'Halazzi - SAY_PLAYER_KILL'),
+(23577,1,1,'Ya all gonna fail!',14,0,100,0,0,12027,0,0,'Halazzi - SAY_PLAYER_KILL'),
+(23577,2,0,'Me gonna carve ya now!',14,0,100,0,0,12023,0,0,'Halazzi - SAY_MELEE'),
+(23577,2,1,'You gonna leave in pieces!',14,0,100,0,0,12024,0,0,'Halazzi - SAY_MELEE'),
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1670753866467974200.sql:8]: In Core this is SAY_SABER, update the comment accordingly.

File: data/sql/updates/pending_db_world/rev_1670753866467974200.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+--
+DELETE FROM `creature_text` WHERE `CreatureID`=23577;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(23577,0,0,'Get on ya knees and bow.... to da fang and claw!',14,0,100,0,0,12020,0,0,'Halazzi - SAY_AGGRO'),
+(23577,1,0,'You can\t fight da power!',14,0,100,0,0,12026,0,0,'Halazzi - SAY_PLAYER_KILL'),
+(23577,1,1,'Ya all gonna fail!',14,0,100,0,0,12027,0,0,'Halazzi - SAY_PLAYER_KILL'),
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1670753866467974200.sql:6]: In Core this is SAY_KILL, update the comment accordingly.

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by UltraNix** [comment]
@Kitzunu Done.

**Review by Kitzunu — APPROVED** [review]

**Comment by Maelthyrr** [comment]
Hi, tested and work fine :) Thanks.

**Pull request merged**